draft-ietf-jose-hpke-encrypt.md

```diff
@@ -261,20 +261,24 @@ After verification:
 # Key Encryption
 
 HPKE based recipients can be added alongside existing `ECDH-ES+A128KW` or `RSA-OAEP-384` recipients because HPKE is only used to encrypt the content encryption key, and because the protected header used in content encryption is passed to HPKE as Additional Authenticated Data.
+This protected header is formatted as BASE64URL(UTF8(JWE Protected Header)).
```

Comment:
```suggestion
```

This sentence is redundant.  Please delete it.  This is already defined by JWE.

Reply:
It was to address the comment from Illari on "How is HPKE AAD input set"
I can re-phrase the line to say 
The protected header encoding remains consistent with existing JWE formatting rules.